study-builder/src/main/java/org/broadinstitute/ddp/studybuilder/StudyBuilderCli.java

```diff
@@ -146,6 +149,15 @@ private void run(String[] args) throws Exception {
             execute(builder::runWorkflow, isDryRun);
             log("done");
             return;
+        } else if (cmd.hasOption("only-recreate-workflow")) {
+            if (studyHasPatch(cfgPath) && !promptForConfirmation("There are patches for this study. Have you checked that workflow DOES "
+                    + "NOT refer to any activities added in patches for this study?")) {
+                return;
+            }
```

Comment:
Based on Slack exchange with @yufengwng , just adding a confirmation step. Guess we won't be able to use this option every time if there are activities created in patches that are referenced by workflow, but can be used sometimes, like for Osteo.